mne/time_frequency/tests/test_csd.py

```diff
@@ -161,3 +163,163 @@ def test_compute_epochs_csd_on_artificial_data():
                     delta = 0.004
                 assert_true(abs(signal_power_per_sample -
                                 mt_power_per_sample) < delta)
+
+
+def test_compute_csd():
+    """Test computing cross-spectral density from ndarray
+    """
```

Comment:
Turns out this should all be on one line `"""Test ... """`. Feel free to change the other instances in the file if you want, which I assume is what led you astray.